shift_detector/checks/WordPredictionCheck.py

```diff
@@ -41,9 +39,7 @@ def is_numeric(value):
     def run(self, store) -> Report:
 
         if self.columns is None:
-            df1_columns = set(store[ColumnType.text][0].columns)
-            df2_columns = set(store[ColumnType.text][1].columns)
-            self.columns = list(df1_columns.intersection(df2_columns))
+            self.columns = store[ColumnType.text]
```

Comment:
This does not work. Should be `store[ColumnType.text][0].columns`. How about a test? 😉